src/renderers/webgl/WebGLBackground.js

```diff
@@ -25,7 +25,7 @@ function WebGLBackground( renderer, state, objects, premultipliedAlpha ) {
 
 	function render( renderList, scene, camera, forceClear ) {
 
-		let background = scene.isScene === true ? scene.background : null;
+		let background = scene.isScene === true ? textures.getCubeTexture( scene.background ) : null;
```

Comment:
`scene.background` can be also an instance of `THREE.Color`. Besides, an instance of `THREE.Texture` can represent and normal (flat) background texture. In both cases, `getCubeTexture()` should not be called.

Reply:
Ah, good catch.